application-services/custom/influxdb-export/res/configuration.toml

```diff
@@ -0,0 +1,59 @@
+[Writable]
+  LogLevel = "INFO"
+
+[Service]
+BootTimeout = "30s"
+CheckInterval = "10s"
+Host = "localhost"
+ServerBindAddr = "" # if blank, uses default Go behavior https://golang.org/pkg/net/#Listen
+Port = 50002
+Protocol = "http"
+ReadMaxLimit = 100
+StartupMsg = "App service export to InfluxDB has Started"
+Timeout = "5s"
+
+[Registry]
+Host = "192.168.0.35"
+Port = 8500
+Type = "consul"
+
+[Clients]
+  [Clients.CoreData]
+  Protocol = "http"
+  Host = "192.168.0.35"
+  Port = 48080
+
+  [Clients.Logging]
+  Protocol = "http"
+  Host = "192.168.0.35"
+  Port = 48061
+
+[Binding]
+Type="messagebus"
+SubscribeTopic="events"
+
+[MessageBus]
+Type = "zero"
+    [MessageBus.SubscribeHost]
+        Host = "192.168.0.35"
+        Port = 5563
+        Protocol = "tcp"
+
+[Logging]
+EnableRemote = false
+File = ""
+
+[ApplicationSettings]
+BrokerAddress = "tcp://192.168.0.33:1883"
+Publisher = ""
+Topic = "edgex/EdgeXEvents"
+
+[Writable.InsecureSecrets]
+  [Writable.InsecureSecrets.mqtt]
+      path = "mqtt"
+      [Writable.InsecureSecrets.mqtt.Secrets]
+          username = "edgex"
+          password = "password"
+          cacert = ""
+          clientcert = ""
+          clientkey = ""
```

Comment:
This section should be up a top under the **[Writable]** section. Interesting that it actually works here...

```
[Writable]
  LogLevel = "INFO"
  [Writable.InsecureSecrets]
     [Writable.InsecureSecrets.mqtt]
        path = "mqtt"
        [Writable.InsecureSecrets.mqtt.Secrets]
            username = "edgex"
            password = "password"
            cacert = ""
            clientcert = ""
            clientkey = "" 
```

Reply:
I am guessing that it is just namespaced and the upper level [ ]'s only help setup the empty namespace.  But I'll add it to make sure there are no issues.

Reply:
I is more of a consistency thing with other TOML files and ease of find all things writable...